python/lsst/analysis/tools/actions/plot/gridPlot.py

```diff
@@ -65,14 +68,24 @@ class GridPlot(PlotAction):
         doc="Number of columns.",
         default=1,
     )
+    width_ratios = ListField[float](
+        doc="width ratios",
+        optional=True,
+    )
+    height_ratios = ListField[float](
+        doc="height ratios",
+        optional=True,
+    )
     xDataKeys = DictField[int, str](
-        doc="Dependent data definitions. The key of this dict is the panel ID. The values are keys of data "
-        "to plot (comma-separated for multiple) where each key may be a subset of a full key.",
+        doc="Dependent data definitions. The key of this dict is the panel ID. "
+        "The values are keys of data to plot (comma-separated for multiple) where "
+        "each key may be a subset of a full key.",
         default={},
     )
     valsGroupBy = DictField[int, str](
-        doc="Independent data definitions. The key of this dict is the panel ID. The values are keys of data "
-        "to plot (comma-separated for multiple) where each key may be a subset of a full key.",
+        doc="Independent data definitions. The key of this dict is the panel ID. "
+        "The values are keys of data to plot (comma-separated for multiple) where "
+        "each key may be a subset of a full key.",
```

Comment:
Other than moving the sentence to a new line, did something else change here?

Reply:
No, I believe this was just a black edit.

Reply:
Hm, interesting, black doesn't change this for me. At least, not in VSCode with this in my `settings.json`:
```
    "black-formatter.args": [
        "--line-length",
        "110",
    ],
```

Reply:
This is fixed now - keeping the longer lines. Don't know what happened there!